catalyst/dl/experiment/config.py

```diff
@@ -319,6 +321,35 @@ def get_scheduler(self, stage: str, optimizer: Optimizer) -> Scheduler:
         )
         return scheduler
 
+    @staticmethod
+    def _get_transform(**params) -> Callable:
+        if "transforms" in params:
+            transforms_composition = [
+                ConfigExperiment._get_transform(**transform_params)
+                for transform_params in params["transforms"]
+            ]
+            params.update(transforms=transforms_composition)
+
+        transform = TRANSFORMS.get_from_params(**params)
+
+        return transform
+
+    def get_transforms(self, stage: str = None, mode: str = None) -> Callable:
+        transform_params = deepcopy(
```

Comment:
🤔 could you please add `_key_value` logic, so we could sync the transformers for different key by [Augmentor](https://github.com/catalyst-team/catalyst/blob/master/catalyst/data/augmentor.py#L4)?

Reply:
moreover, we could also add TupleAugmentor to handle tuple-like bathes, like MNIST and CIFAR one

Reply:
I'm not sure I follow you, could you please provide an example of `_key_value` for `transform_params`?
Do you mean something like this
```
transform_params:
  _key_value: True

  image:
    transform: ...
  mask:
    transform: ...
  keypoint:
    transform: ...
```
?

Reply:
yes, just like that